inlang/source-code/paraglide/paraglide-js/package.json

```diff
@@ -8,7 +8,8 @@
 	},
 	"exports": {
 		"./*/messages": "./dist/compiled-output/*/messages.js",
-		"./*": "./dist/compiled-output/*/runtime.js"
+		"./*": "./dist/compiled-output/*/runtime.js",
+		".": "./dist/index.js"
```

Comment:
oh oh, not sure if this works with no namespace imports. needs to be tested

Reply:
I tested this code already with the sveltekit adpater example no problems. where is the problem?